Makefile

```diff
@@ -1,13 +1,23 @@
 GOPATH ?= $(shell go env GOPATH)
 
+.PHONY: build
+build:
+	mkdir -p bin
+	GOBIN="$(CURDIR)/bin/" go install ./cmd/...
+
```

Comment:
This target wasn't changed.  I just moved it to the top making it the default target.